### PR TITLE
Fixes for 'Possible External RDP Brute-Force - Set Verdict' XSIAM pla…

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10808

## Description
As part of the creation of test playbooks for XSIAM 'Core; pack effort, I reviewed 'Possible External RDP Brute-Force - Set Verdict' and noticed the following issue:
There is a mapping rule configured for an alert field that doesn't exist named 'XDR Alert Search Results' in the task 'Set related alerts'.

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-Possible_External_RDP_Brute-Force_-_Set_Verdict.yml [hunk 1]
```diff
@@ -205,7 +205,7 @@ tasks:
             iscontext: true
           right:
             value:
-              simple: "false"
+              simple: "true"
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-Possible_External_RDP_Brute-Force_-_Set_Verdict.yml:208]: Please change it back to 'false'.

File: Packs/Core/ReleaseNotes/3_0_38.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Possible External RDP Brute-Force - Set Verdict
+
+- Fixed an issue where the *'Connection from unusual country?'* conditional task checked whether the *'UnusualGeoLocation'* playbook input equaled 'false' instead of 'true'.
```
Comment (OmriItzhak) [inline Packs/Core/ReleaseNotes/3_0_38.md:6]: Please remove this line.

File: Packs/Core/Playbooks/playbook-Possible_External_RDP_Brute-Force_-_Set_Verdict.yml (hunk 1 above)
Comment (TalNos) [reply Packs/Core/Playbooks/playbook-Possible_External_RDP_Brute-Force_-_Set_Verdict.yml:208]: done

File: Packs/Core/ReleaseNotes/3_0_38.md (hunk 2 above)
Comment (TalNos) [reply Packs/Core/ReleaseNotes/3_0_38.md:6]: done

**Review by OmriItzhak — APPROVED** [review]

**Pull request merged**